### DISCUSSION TITLE
Access Control Centre Items

### DISCUSSION THREAD

**Discussion by basbebe** [FelixKratz/SketchyBar#157] [Q&A] [answered]
I'm trying out SketchyBar and started with the setup shared by @FelixKratz [here](https://github.com/FelixKratz/SketchyBar/discussions/47#discussion-3587958).

I can't seem to get the elements from Control Centre displayed.

![Screenshot 2022-01-16 at 12 18 06](https://user-images.githubusercontent.com/7083188/149657700-b3c16ad8-292a-406e-abec-4a824d73cdb3.png)

I don't get any errors running sketchybar from commandline:
```fish
❯ sketchybar
sketchybar configuration loaded..
```

I gave Screen Recording Permissions.

Where would I start to find out what's happening?

I'm using Monterey 12.1 set to english

**Comment by basbebe** [comment]
I just realized that my system language was set to "English", not "English (US)".

So this might be related to the centre / center situation.

Will check later.

**Comment by FelixKratz** [comment, marked as answer]
You can check if those items I have aliased in the config actually exist on your system:
```bash
sketchybar --query default_menu_items
```
Probably your Control Center is not called `Control Center` but `Control Centre` which you would need to change.

**Reply by basbebe** [reply]
Thanks!

So with Language set to 'English (US)', it is called `Control Center` and it works now!

Will probably keep it out for the moment though since I can't change its color (obviously)

**Reply by FelixKratz** [reply]
You can change the color of aliases since 657a2b0798472a3e232bb648e0d5cc48c32af1d7:
```bash
sketchybar --set <name> alias.color=<argb_hex>
```

**Reply by basbebe** [reply]
Thanks, can't wait to try this out!
